camera.py

```diff
@@ -0,0 +1,26 @@
+import pygame.camera
+import pygame.event as pg_event
+
+resolution = (1280, 720)
+camera_path = "/dev/video0"
+
+
+def camera(camera_path):
```

Comment:
Instead of creating global variables, you can do something like:

```python
def camera(path='/dev/video0', resolution=(1280, 720)):
```

Reply:
Yes, it is a good idea